tracing-attributes/src/attr.rs

```diff
@@ -6,6 +6,12 @@ use quote::{quote, quote_spanned, ToTokens};
 use syn::ext::IdentExt as _;
 use syn::parse::{Parse, ParseStream};
 
+#[derive(Clone, Default, Debug)]
```

Comment:
nit, not a big deal: it might be nice to have a comment on this like
```suggestion
/// Arguments to `#[instrument(err(...))]` and `#[instrument(ret(...))]` which
/// describe how the return value event should be emitted.
#[derive(Clone, Default, Debug)]
```

Reply:
Done.